src/pages/Edit/Edit.tsx

```diff
@@ -100,7 +100,14 @@ const Edit = (): JSX.Element => {
       return true;
     });
 
-    filteredIncomes && setDisplayedIncome(filteredIncomes);
+    const sortIncomes = filteredIncomes.sort((a,b) => {
+      const firstPerson = a.surname ? a.surname : 'ŹŹŹ';
```

Comment:
Why we use this string 'ŹŹŹ'?

Reply:
if the element missing surname (undefined), i put it last element in sort.